### DISCUSSION TITLE
RDS나 Aurora를 쓰면 일반적으로 스레드풀 지원이 되지 않는걸까?

### DISCUSSION THREAD

**Discussion by CODe5753** [Well-Founded-Confidence/RealMySQL#17] [Q&A]
> MySQL 서버 엔터프라이즈 에디션은 해당 기능을 제공하지만, 커뮤니티 에디션은 해당 기능을 지원하지 않는다

이런 내용이 있었는데요, 그럼 프로덕션에서 쓰이는 RDS, Aurora 같은 환경에서도 안쓰이는걸까요?

**Comment by menthamin** [comment]
RDS 인스턴스 생성시: 버전 선택이 있을 수 있다
